### DISCUSSION TITLE
请教下是否有参数控制可以不打印access log

### DISCUSSION THREAD

**Discussion by hoocoo** [higress-group/higress#834] [Q&A]
请教下是否有参数控制可以不打印access log，在文档没查到

**Comment by johnlanni** [comment]
![image](https://github.com/alibaba/higress/assets/6763318/b45ee080-d021-4b8f-97de-597a9ef3bd88)
这个复用的istio的配置，这里配成空字符串就可以了

kubectl edit cm -n higress-system higress-config

**Reply by johnlanni** [reply]
参考：https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/
